core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -1154,7 +1156,7 @@ public void testCommitAdvancesMinTranslogForRecovery() throws IOException {
 
     public void testSyncedFlush() throws IOException {
         try (Store store = createStore();
-            Engine engine = new InternalEngine(config(defaultSettings, store, createTempDir(), new LogByteSizeMergePolicy(), null))) {
+             Engine engine = new InternalEngine(config(defaultSettings, store, createTempDir(), new LogByteSizeMergePolicy(), null))) {
```

Comment:
well, it's part of the try() clause and not the body, I think this is good ? (it's also the auto formatter)